### DISCUSSION TITLE
The documentation for ExifData doesn't match the current release (thumbnail from EXIF)

### DISCUSSION THREAD

**Discussion by jrmoore** [dlemstra/Magick.NET#998] [General] [answered]
I'd like to extract the thumbnail embedded in the EXIF data of an image if present, which led me to its page in the documentation, from https://github.com/dlemstra/Magick.NET/blob/main/docs/ExifData.md :

    // Read image from file
    using (var image = new MagickImage("FujiFilmFinePixS1Pro.jpg"))
    {
        // Retrieve the exif information
        var profile = image.GetExifProfile();
    
        // Create thumbnail from exif information
        using (var thumbnail = profile.CreateThumbnail())
        {
            // Check if exif profile contains thumbnail and save it
            if (thumbnail != null)
                thumbnail.Write("FujiFilmFinePixS1Pro.thumb.jpg");
        }
    }

Although that code used to work previously, the CreateThumbnail() method is not part of IExifProfile interface anymore, what would be the current way of doing it?

**Comment by dlemstra** [comment, marked as answer]
The `CreateThumbnail` method is an extension method and no longer available if you need to use .NET Framework instead of .NET Standard. If you can use extension methods in your project you could copy this into your project: https://github.com/dlemstra/Magick.NET/blob/eff44bf236fcd92b20ec518e186006939dd1169c/src/Magick.NET/Netstandard20/Extensions/IExifProfileExtensions.cs

**Reply by jrmoore** [reply]
Thanks! :)
